mobile/src/navigations/main/main.tsx

```diff
@@ -1,30 +1,85 @@
-import {
-  type BottomTabNavigationOptions,
-  createBottomTabNavigator,
-} from '@react-navigation/bottom-tabs';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { getHeaderTitle } from '@react-navigation/elements';
+import { type RouteProp } from '@react-navigation/native';
 import React from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
 
+import ChatIcon from '#assets/img/icons/chat.svg';
 import HomeIcon from '#assets/img/icons/home.svg';
 import { AppColor, MainScreenName } from '#libs/enums/enums';
-import { type TabNavigationParameterList } from '#libs/types/types';
-import { Home } from '#screens/main/home';
+import {
+  type TabNavigationParameterList,
+  type TabNavigatorOptions,
+} from '#libs/types/types';
+import { Chat } from '#screens/chat/chat';
+import { ChatHeader } from '#screens/chat/components/components';
+import { HomeHeader } from '#screens/home/components/components';
+import { Home } from '#screens/home/home';
 
 import { styles } from './styles';
 
 const BottomTab = createBottomTabNavigator<TabNavigationParameterList>();
 
-const tabNavigatorOptions: BottomTabNavigationOptions = {
-  tabBarActiveTintColor: AppColor.BLUE_200,
-  tabBarIcon: ({ color }) => <HomeIcon color={color} />,
-  tabBarInactiveTintColor: AppColor.GRAY_300,
-  tabBarShowLabel: false,
-  tabBarStyle: styles.tabBarStyle,
-};
-
 const Main: React.FC = () => {
+  const tabNavigatorOptions = ({
+    route,
+  }: {
+    route: RouteProp<TabNavigationParameterList>;
+  }): TabNavigatorOptions => ({
+    tabBarActiveTintColor: AppColor.BLUE_300,
+    tabBarIcon: ({ color }: { color: string }): React.ReactNode => {
+      switch (route.name) {
+        case MainScreenName.HOME: {
+          return <HomeIcon color={color} />;
+        }
+        case MainScreenName.CHAT: {
+          return <ChatIcon color={color} />;
+        }
+      }
+    },
```

Comment:
We can add tabBarIcon to the options property in BottomTab.Screen to avoid this logic here

Reply:
Then you will not need route, so you can create tabNavigatorOptions as an object

Reply:
Done